l10n_es_facturae/models/account_invoice.py

```diff
@@ -71,6 +75,24 @@ class AccountInvoice(models.Model):
         inverse_name='invoice_id',
         copy=False
     )
+    facturae_start_date = fields.Date()
+    facturae_end_date = fields.Date()
+
+    @api.constrains('facturae_start_date', 'facturae_end_date')
+    def _check_facturae_date(self):
+        for record in self:
+            if bool(record.facturae_start_date) != bool(
+                record.facturae_end_date
+            ):
+                raise ValidationError(_(
+                    'Start date is required if you define end date'
```

Comment:
El mensaje debería ser `FacturaE start and end dates are both required if one of them is filled.`